gcp/bootstrap.tf

```diff
@@ -0,0 +1,260 @@
+#  deploy image
+resource "google_compute_instance" "bootstrap" {
+   name         = "${data.template_file.cluster-name.rendered}-bootstrap"
+   machine_type = "${var.gcp_bootstrap_instance_type}"
+   zone         = "${data.google_compute_zones.available.names[0]}"
+ 
+  labels {
+   owner = "${coalesce(var.owner, data.external.whoami.result["owner"])}"
+   expiration = "${var.expiration}"
+   name = "${data.template_file.cluster-name.rendered}-bootstrap"
+   cluster = "${data.template_file.cluster-name.rendered}"
+  }
+
+  boot_disk {
+    initialize_params {
+      image = "${module.dcos-tested-gcp-oses.gcp_image_family}/${module.dcos-tested-gcp-oses.gcp_image_name}"
+      size  = "${var.instance_disk_size}"
+    }
+  }
+
+  network_interface {
+    subnetwork = "${google_compute_subnetwork.private.name}"
+    access_config {
+        // IP
```

Comment:
probably should remove this comment or add a meaningful message

Reply:
done! tfvm